UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1625,7 +1709,14 @@ public void ServerCreateSprite(BodyPart implant)
 		{
 			int i = 0;
 			bool isSurfaceSprite = implant.IsSurface || implant.BodyPartItemInheritsSkinColor;
-			var sprites = implant.GetBodyTypeSprites(playerSprites.ThisCharacter.BodyType);
+
+			BodyType bodyType = BodyType.NonBinary;
+			if (playerSprites.ThisCharacter != null)
```

Comment:
if this character is null, something is not right. I think we should log this.

Reply:
cyborgs don't have character settings